src/de.rs

```diff
@@ -1882,12 +1889,11 @@ impl<'de, R: Read<'de>> de::Deserializer<'de> for &mut Deserializer<R> {
                         self.eat_char();
                         Ok(value)
                     }
-                    Some(_) => Err(self.error(ErrorCode::ExpectedSomeValue)),
+                    Some(_) => Err(self.error(ErrorCode::ExpectedObjectCommaOrEnd)),
```

Comment:
Nit pick: the error code isn't 100% correct. It states that `,` was expected `}`. However, only `}` is valid here.

Here's an example of that
```rust
#[derive(Deserialize, Debug)]
enum Foo {
    Bar(u64, bool),
}

let data = r#"{"Bar": [1234, true],}"#;
let res = from_str::<Foo>(data);
dbg!(res);
```
Yields
```
res = Err(
    Error("expected `,` or `}`", line: 1, column: 20),
)
```
So the error is complaining that the `,` isn't a `,` or `}` :upside_down_face:

Reply:
Good point. Fixed